roles/network_plugin/weave/templates/weave-net.yml.j2

```diff
@@ -67,16 +67,16 @@ items:
     kind: Role
     metadata:
       name: weave-net
+      namespace: kube-system
       labels:
         name: weave-net
-      namespace: kube-system
```

Comment:
Why this changes the order of the namespace?
The previous order is the same as the original manifest.

Reply:
I kept the ordering from the original file and just added in the various Ansible bits from kubespray. I'm happy to change the ordering to make the PR have less changes.

Same applies to your additional comments on ordering below.